### PR TITLE
Add handling multiple handle_caps callbacks

### PR DESCRIPTION
**Pull request by WojciechBarczynski**
Improved handling add video to rust VC in `handle_caps`.
Changed `add_video` to `put_video` for updating video properties.

### CODE REVIEW

File: lib/membrane/compositor_element.ex [hunk 1]
```diff
@@ -19,19 +19,21 @@ defmodule Membrane.VideoCompositor.CompositorElement do
...
+      description: "Struct with video width, height, framerate and pixel format."
+    ],
+    real_time: [
+      spec: boolean(),
+      description: """
+      Set the compositor to real-time mode.
+      """,
+      default: false
+    ]
+  )
```
Comment (DominikWolek) [inline lib/membrane/compositor_element.ex:34]: Weird formating

File: lib/membrane/compositor_element.ex [hunk 2]
```diff
@@ -19,19 +19,21 @@ defmodule Membrane.VideoCompositor.CompositorElement do
...
+
+  def_input_pad(:input,
```
Comment (DominikWolek) [inline lib/membrane/compositor_element.ex:36]: Weird formating here too, please fix this

**Review by DominikWolek — APPROVED** [review]

File: lib/membrane/compositor_element.ex (hunk 1 above)
Comment (WojciechBarczynski) [reply lib/membrane/compositor_element.ex]: fixed

File: lib/membrane/compositor_element.ex (hunk 2 above)
Comment (WojciechBarczynski) [reply lib/membrane/compositor_element.ex]: fixed

**Pull request merged**